src/libraries/Microsoft.PowerFx.Core/Lexer/TexlLexer.cs

```diff
@@ -1567,7 +1572,7 @@ private Token LexStringLit()
                         _sb.Append(ch);
                         NextChar();
                     }
-                    else if (!CharacterUtils.IsFormatCh(ch))
+                    else if (_pfxV1 || !CharacterUtils.IsFormatCh(ch))
```

Comment:
>_pfxV1 || !CharacterUtils.IsFormatCh(ch) [](http://example.com/codeflow?start=29&length=40)

nit - since this is repeated multiple times, consider an instance method:

bool IncludeCharInLiteral(ch) {
// v1 always includes all chars.
// pre-v1 would excluside certain formatting chars from string literals. 
return _pfxV1 || !CharacterUtils.IsFormatCh(ch)

}